frontend/ui/geometry.lua

```diff
@@ -208,6 +208,32 @@ function Geom:intersectWith(rect_b)
     return not self:notIntersectWith(rect_b)
 end
 
+--[[--
+Returns true if self does not share any area with rect_b
+
+@tparam Geom rect_b
+]]
+function Geom:notOpenIntersectWith(rect_b)
```

Comment:
What does "open" means in this context ? If it is about the edges being shared, "open" does not intuitively relate to that for me :)

Reply:
Half-open range, e.g., `[x, x + w)` instead of [x, x+ w]`.